packages/eas-cli/src/commands/update/index.ts

```diff
@@ -143,6 +145,10 @@ export default class UpdatePublish extends EasCommand {
       description: `File containing the PEM-encoded private key corresponding to the certificate in expo-updates' configuration. Defaults to a file named "private-key.pem" in the certificate's directory.`,
       required: false,
     }),
+    'no-channel': Flags.boolean({
+      description: `Skip EAS channel creation.`,
+      default: false,
+    }),
```

Comment:
maybe we can do
```suggestion
    'no-channel': Flags.boolean({
      description: `Skip EAS channel creation.`,
      default: false,
      exclusive: ['channel'],
    }),
```
?